### PR TITLE
Update 3.22

### PR DESCRIPTION
**Pull request by SilentGert**
Added update notes for version 3.22 and updated the compatibility list and version number.

### CODE REVIEW

File: docs/update-notes.rst [hunk 1]
```diff
@@ -9,6 +9,80 @@ It lists changes to technical requirements for new versions and additional instr
...
+	To update Workflow Accelerator from a version prior to 3.11, you MUST first update the system to version 3.15 before you can update to version 3.22. 
+	Download `Signavio Workflow 3.15 <https://download.signavio.com/onpremise/latest/workflow/signavio-workflow-v3.15.11.zip>`_ and follow the respective previous update instructions, before you continue with this section.
+
+Besides the application update you are also advised to update your MongoDB instance.
+See the following section for more information.
+
+Updating MongoDB
+````````````````
+Workflow Accelerator 3.22 now supports the latest version of MongoDB (3.4.x) which comes with improved performance and functionality.
+You are advised to update your instance of MongoDB to version 3.4.
```
Comment (hilton) [inline docs/update-notes.rst]: _You are advised to_ sounds vague and uncertain: do users have to do it or not. I suggest:

> You should upgrade to MongoDB version 3.4, as Workflow Accelerator releases will require it.

(and possibly remove the next sentence)

File: docs/update-notes.rst [hunk 2]
```diff
@@ -9,6 +9,80 @@ It lists changes to technical requirements for new versions and additional instr
...
+~~~~~~~
+#. Open a command line with administrative privileges
+#. Navigate to the MongoDB bin directory
+
+	* For example ``C:\MongoDB\bin``
+
+#. Execute the following command:
+
+	* Replace ``<enterYourUserHere>`` with your MongoDB user, e.g. ``signavio``
+	* Replace ``<enterYourPasswordHere>`` with your MongoDB user password
```
Comment (hilton) [inline docs/update-notes.rst]: Isn't it neater to just use `<username>` and `<password>`?

File: docs/update-notes.rst [hunk 3]
```diff
@@ -9,6 +9,80 @@ It lists changes to technical requirements for new versions and additional instr
...
+Updating to Signavio Workflow Accelerator 3.22
+----------------------------------------------
+Signavio Workflow has been renamed to Signavio Workflow Accelerator.
+If you use Workflow Accelerator in combination with Signavio Process Manager, you must update Process Manager to version 10.15.
+
+.. important:: 
+	To update Workflow Accelerator from a version prior to 3.11, you MUST first update the system to version 3.15 before you can update to version 3.22. 
+	Download `Signavio Workflow 3.15 <https://download.signavio.com/onpremise/latest/workflow/signavio-workflow-v3.15.11.zip>`_ and follow the respective previous update instructions, before you continue with this section.
+
+Besides the application update you are also advised to update your MongoDB instance.
```
Comment (hilton) [inline docs/update-notes.rst]: Change to _As well as upgrading the application, you should upgrade MongoDB._

File: docs/update-notes.rst [hunk 4]
```diff
@@ -9,6 +9,80 @@ It lists changes to technical requirements for new versions and additional instr
...
+To update your standalone MongoDB 3.2 to version 3.4 follow these steps:
+
+#. Stop the Tomcat application server - see :ref:`stopping`.
+#. Create a backup of your database using ``mongodump`` - see :ref:`backup`.
+#. Stop MongoDB - see :ref:`stopping`.
+#. *Windows only:* manually remove the created MongoDB service by executing ``$MONGO_HOME\bin\mongod.exe --remove`` with administrator privileges on the command line ``cmd``
+#. Uninstall MongoDB 3.2.x, but ensure you don't delete the data directory and configuration file of your database as well as the backup!
+#. Install MongoDB 3.4.x.
+
+	* You can find a suitable version for your OS in the MongoDB download center: https://www.mongodb.com/download-center
```
Comment (hilton) [inline docs/update-notes.rst]: _OS_ is a gratuitous abbreviation here - write it out as _operating system_

File: docs/update-notes.rst [hunk 5]
```diff
@@ -9,6 +9,80 @@ It lists changes to technical requirements for new versions and additional instr
...
+
+#. *Windows only:* Open the command line ``cmd`` with administrative privileges and execute the following command. Make sure to use absolute paths and replace them with the matching ones on your system.
+
+    * ``C:\MongoDB\bin\mongod.exe --config C:\MongoDB\mongod.cfg --install``
+    * The referenced configuration file is the one you have created previously for the old version of MongoDB and should still be in the same location.
+
+#. Start the new MongoDB instance.
+#. Enable new functionality for version 3.4 by executing the following command on command line
```
Comment (hilton) [inline docs/update-notes.rst]: _… on the command line._

File: docs/update-notes.rst [hunk 6]
```diff
@@ -9,6 +9,80 @@ It lists changes to technical requirements for new versions and additional instr
...
+#. Stop the Tomcat application server - see :ref:`stopping`.
+#. Create a backup of your database using ``mongodump`` - see :ref:`backup`.
+#. Stop MongoDB - see :ref:`stopping`.
+#. *Windows only:* manually remove the created MongoDB service by executing ``$MONGO_HOME\bin\mongod.exe --remove`` with administrator privileges on the command line ``cmd``
+#. Uninstall MongoDB 3.2.x, but ensure you don't delete the data directory and configuration file of your database as well as the backup!
+#. Install MongoDB 3.4.x.
+
+	* You can find a suitable version for your OS in the MongoDB download center: https://www.mongodb.com/download-center
+
+#. *Windows only:* Open the command line ``cmd`` with administrative privileges and execute the following command. Make sure to use absolute paths and replace them with the matching ones on your system.
```
Comment (hilton) [inline docs/update-notes.rst]: _ones_ sounds odd (don't know why) - replace with _paths_

File: docs/update-notes.rst (hunk 2 above)
Comment (SilentGert) [inline docs/update-notes.rst]: Hmm yeah, although this version looks more disruptive and not like it belongs to the command. :D

**Pull request merged**